ll.tm

```diff
@@ -201,6 +201,7 @@ int_type_tok : /i[0-9]+/
 'br' : /br/
 'builtin' : /builtin/
 'byval' : /byval/
+'blockcount': /blockcount/
```

Comment:
sort the list of keyword tokens alphabetically.

Reply:
Maybe use the order the same as it shows in the C++ source code? Then insert a new one can be faster.

Reply:
I prefer to not depend too much on the structure of the C++ code base, at least if possible to avoid. The grammar document should be an independent source of reference as far as possible.

Finding the new tokens to insert should be easy enough with the diff:

```diff
+++ llvm-11.0.0.src/lib/AsmParser/LLToken.h	2020-10-07 12:10:48.000000000 +0200
@@ -196,6 +196,7 @@
   kw_naked,
   kw_nest,
   kw_noalias,
+  kw_noundef,
   kw_nobuiltin,
   kw_nocapture,
   kw_noduplicate,
@@ -204,15 +205,18 @@
   kw_noinline,
   kw_norecurse,
   kw_nonlazybind,
+  kw_nomerge,
   kw_nonnull,
   kw_noredzone,
   kw_noreturn,
   kw_nosync,
   kw_nocf_check,
   kw_nounwind,
+  kw_null_pointer_is_valid,
   kw_optforfuzzing,
   kw_optnone,
   kw_optsize,
+  kw_preallocated,
   kw_readnone,
...
```

If you prefer @dannypsnl, we can just merge this as is, and I can sort it in a follow-up commit?

Reply:
Sure

Reply:
I mean, find inserted position BTW xd.
Then if we have new token, we can search previous token to get where to insert the new one in syntax doc

Reply:
> If you prefer @dannypsnl, we can just merge this as is, and I can sort it in a follow-up commit?

Done.